README.md

```diff
@@ -110,37 +98,16 @@ sudo rm -rf **\~/.solcx**
 sudo rm -rf **\~/.vvm**
 ```
 
-## Environment Variables:
-
-Some plugins require environment variables to connect to their external systems, such project IDs or API keys.
-Follow instructions from individual plugin documentations, such as:
-
-* [ape-alchemy](https://github.com/ApeWorX/ape-alchemy/blob/main/README.md#quick-usage)
-* [ape-infura](https://github.com/ApeWorX/ape-infura#readme)
+## Ape Modular Plugin Sytem:
 
-Generally, set environment variables by doing the following:
+One of the best feature of Ape is our modular plugin system. Our plugins is the way we want you to have most interoperable experience with Web Ecosystem.
```

Comment:
I think this second sentence needs revising. For example, `Our plugins is the way ...` should probably be `Our plugins are the way`, and I don't know what `Web Ecosystem` means or know why it is capitalized.